sourcemash/api/items.py

```diff
@@ -17,18 +16,46 @@
     'category_1': fields.String,
     'category_2': fields.String,
     'summary': fields.String,
-    'category_1': fields.String,
-    'category_2': fields.String,
+    'totalVotes': fields.Integer,
     'uri': fields.Url('api.item')
 }
 
 
 class ItemAPI(Resource):
+    def __init__(self):
+        self.reqparse = reqparse.RequestParser()
+        self.reqparse.add_argument('vote', type = int, required = True)
```

Comment:
PUT on item will also need to handle "mark_as_read" eventually.

We will not want to pass both {read: True} and {vote: 0} every time we mark an article as read. Reqparse can take a default argument if it's not present, so that way we can remove required.

This way a mark_as_read and vote PUT won't have to be handled any different. We just need to specify on line 52 that args.vote != 0 (or check this on the form once we write that).

Does this make sense?